application/common/models/User.php

```diff
@@ -93,10 +94,6 @@ protected function attemptPasswordMigration()
         }
     }
 
-    /**
```

Comment:
I assumed since the type and name of the param are already there in the function, the docs were pretty redundant.  Is that the case or do you needs those docs for something else?  

As for the `@return`, I assume that's very common pattern but if you think it's helpful for the consumer that may not know then I can add it back.